docs/source/index.rst

```diff
@@ -98,12 +98,12 @@ Then add the following line, replacing ``api_key_here`` with your key::
 
 **Linux**
 
-Open ``.bash_profile`` in your favorite editor (nano is used here):
+Open ``.bashrc`` in your favorite editor (nano is used here):
 
 .. code-block:: bash
 
    cd ~/
-   nano .bash_profile
+   nano .bashrc
```

Comment:
we mention `.bash_profile` on the README, so we should make this consistent with that.  I don't have a preference for one or the other.

Reply:
Actually, if this is specific to Linux it's fine if the README talks about .bash_profile (which might be more common on Mac?)